### PR TITLE
Feature/add role list and item

### PR DESCRIPTION
**Pull request by Leeyaoi**

### CODE REVIEW

File: src/components/RoleItem/RoleItem.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import MenuButton from "../../shared/UI/MenuButton/MenuButton";
+
+interface Props {
+  user: UserType;
+  role: RoleType;
+}
+
+const RoleItem = ({ user, role }: Props) => {
+  const { deleteRole, currentRole, currentUserId } = useGlobalStore();
+  const getRoleName = () => {
```
Comment (v12code) [inline src/components/RoleItem/RoleItem.tsx]: remove this method. It also menthioned in one of the previous prs

**Review by v12code — APPROVED** [review]

File: src/components/RoleItem/RoleItem.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "../../index.scss";
+
+.RoleItem{
+    width: 100%;
+    border: 1px solid $light-color;
```
Comment (MaliyOna) [inline src/components/RoleItem/RoleItem.scss]: don`t use px

File: src/modules/RoleListModule/RoleListModule.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import "../../index.scss";
+
+.RoleListModule{
+    padding: 0;
+    list-style-type: none;
+    border: 1px solid $light-color;
```
Comment (MaliyOna) [inline src/modules/RoleListModule/RoleListModule.scss]: don`t use px

File: src/modules/RoleListModule/RoleListModule.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    background-color: $back-color;
+    height: 100vh;
+    overflow: auto;
+    width: 100%;
+}
+
+.RoleListModule_RoleList{
+    height: 82%;
+    border: 1px solid $light-color;
```
Comment (MaliyOna) [inline src/modules/RoleListModule/RoleListModule.scss]: don`t use px

**Review by MaliyOna — CHANGES_REQUESTED** [review]

**Review by MaliyOna — APPROVED** [review]

**Pull request merged**